### ISSUE TITLE
Final Issues Before We Go Live!

### ISSUE THREAD

**Issue by johnsiwicki** [bs-production/sure-dry-template#14]
- [x]   https://www.suredrybasements.com/about-us.html   - Mobile layout is busting out of the view port 

- [x]   Can we make the top banner a little larger 
<img width="371" alt="Screen Shot 2021-09-23 at 5 07 49 PM" src="https://user-images.githubusercontent.com/1509641/134584201-b2acfdf9-e087-48aa-a066-0900fe91dcca.png">

 - [x] Phone number should be clickable in the header

- [x]  added this badge to the homepage?? 

```
    <a href="http://www.greatplacetowork.com/certified-company/7022221" title="Rating and Review" target="_blank">
                                <img class="greatplace" src="https://cdn.treehouseinternetgroup.com/cms_images/101/greatplace.png" width="200" height="341" alt="Great Place To Work Certified 2021">
                            </a>
```

- [x]  Fix Reviews Page On Homepage
- [x] Add SVG Icons to Homepage 
- [ ]

**Comment by IamStephan** [comment]
For this:
```html
<a href="http://www.greatplacetowork.com/certified-company/7022221" title="Rating and Review" target="_blank">
 <img class="greatplace" src="https://cdn.treehouseinternetgroup.com/cms_images/101/greatplace.png" width="200" height="341" alt="Great Place To Work Certified 2021">
</a>
```
Maybe we can put it here:
![image](https://user-images.githubusercontent.com/36214317/134587557-19c9d2f6-489f-4e4a-8d9c-aa1668e402c9.png)

with these logos accompanying it (Maybe have a carousel):
![image](https://user-images.githubusercontent.com/36214317/134587699-8733fd83-4ea7-4090-8352-553a5570ace8.png)

**Comment by IamStephan** [comment]
> Can we make the top banner a little larger


Do you mean height-wise, just tweaking the spacing, or removing a section? Or something like this?

![image](https://user-images.githubusercontent.com/36214317/134590546-7df2e9fa-3ec5-41ab-b80d-586adf832431.png)

**Comment by IamStephan** [comment]
> https://www.suredrybasements.com/about-us.html - Mobile layout is busting out of the view port

There are two things at fault here:

1. Two of the widgets on the about page are not mobile optimized. They are:
**Meet The Team Widget**
**Testimonials Widget**
When those two widgets are removed the about page goes to an almost working state
2. When the widgets above have been removed the styles for the `.row` have this weird x padding that almost pushes the content outside the viewport but doesn't cause an overflow (for devices with the same or bigger width than an iphone x, it looks fine). Looks like these styles are added to the HTML from somewhere (Maybe another widget?)...

I tried playing with the article tag to see if it would make the two widgets not jump the viewport, but it doesn't seem to work...
